skimage/morphology/__init__.py

```diff
@@ -1,3 +1,11 @@
+"""Utilities that process images based on shapes.
+
+They are particularly suited for binary images where pixels are represented as 0 or 1,
```

Comment:
I don't think we need to specify that binary images have pixels which can take only two possible values.

Reply:
You are right.